house/admin.py

```diff
@@ -1,5 +1,20 @@
 from django.contrib import admin
 
-from house.models import House
+from house.models import House, City, Country
 
-admin.site.register(House)
+
+class CountryAdmin(admin.ModelAdmin):
+    list_display = ["name"]
```

Comment:
it needs to be in a list because the only field is "name" the tuple doesnt apply, it could be a list at HouseAdmin and CityAdmin (a tuple is a list behind the curtains shh.... )